build.sh

```diff
@@ -30,4 +30,4 @@ check_db () {
 check_disk_space
 pip3 install -e '.[vectors]'
 check_db
-snakemake --resources 'ram=30' -j 2 $@
+snakemake --resources 'ram=30' -j$(nproc) $@
```

Comment:
It is better to use `nproc --ignore=1` so that when the `build.sh` is run there is still some cpu available to do stuff.